### ISSUE TITLE
Checking Firewall on Monitor Ports fails

### ISSUE THREAD

**Issue by dmesser** [ceph/ceph-ansible#754]
Running with ansible 2.0.1.0 on RHEL7.2 the following tasks fails when check_firewall is set to true:

ceph.ceph-common: check if monitor port is not filtered
fatal: [ceph-node-a-1]: FAILED! => {"failed": true, "msg": "'ansible.vars.hostvars.HostVars object' has no attribute u'groups.mons'"}
fatal: [ceph-node-a-2]: FAILED! => {"failed": true, "msg": "'ansible.vars.hostvars.HostVars object' has no attribute u'groups.mons'"}
fatal: [ceph-node-a-3]: FAILED! => {"failed": true, "msg": "'ansible.vars.hostvars.HostVars object' has no attribute u'groups.mons'"}

Disabling check_firewall avoids the issue.

**Comment by leseb** [comment]
do you mind giving this one another try?

**Comment by ksingh7** [comment]
@leseb hitting this 

```
TASK [ceph.ceph-common : include] **********************************************
included: /usr/share/ceph-ansible/roles/ceph-common/tasks/./checks/check_firewall.yml for rhcs2, rhcs1, rhcs3

TASK [ceph.ceph-common : check if nmap is installed] ***************************
ok: [rhcs1]
ok: [rhcs3]
ok: [rhcs2]

TASK [ceph.ceph-common : inform that nmap is not present] **********************
skipping: [rhcs2]
skipping: [rhcs3]
skipping: [rhcs1]

TASK [ceph.ceph-common : check if monitor port is not filtered] ****************
fatal: [rhcs3]: FAILED! => {"failed": true, "msg": "'ansible.vars.hostvars.HostVars object' has no attribute u'groups.mons'"}
fatal: [rhcs1]: FAILED! => {"failed": true, "msg": "'ansible.vars.hostvars.HostVars object' has no attribute u'groups.mons'"}
fatal: [rhcs2]: FAILED! => {"failed": true, "msg": "'ansible.vars.hostvars.HostVars object' has no attribute u'groups.mons'"}
	to retry, use: --limit @site.retry

PLAY RECAP *********************************************************************
rhcs1                      : ok=5    changed=0    unreachable=0    failed=1
rhcs2                      : ok=5    changed=0    unreachable=0    failed=1
rhcs3                      : ok=5    changed=0    unreachable=0    failed=1

```
Temp fix check_firewall: false

**Comment by leseb** [comment]
Smells like an issue with your ansible version.

**Comment by ksingh7** [comment]
NP i will give a try with newer ansible version

**Comment by ASBishop** [comment]
@leseb Do you really mean 2.0.0.2?

I'm running an unregistered RHEL 7.2 and used install-ansible.sh. The script installed epel-release-latest-7, which in turn gave me ansible 2.0.2.0. And that version causes the bug described above.

I want to understand what versions of ansible are not supported because they are too new.

**Comment by matthewrees** [comment]
This will be fixed by #844
